code/mob/living/critter/ai/flock/flocktasks.dm

```diff
@@ -1197,6 +1221,7 @@ butcher
 		holder.target = get_turf(src.target)
 
 /datum/aiTask/timed/targeted/flockdrone_shoot/targetable
+
```

Comment:
?

Reply:
I was gonna put increased targetting range in this one two and then I realised that it wouldn't work because shoot does it differently, so I removed it - accidental white-space was the result.

Reply:
I was gonna put increased targetting range in this one two and then I realised that it wouldn't work because shoot does it differently, so I removed it - accidental white-space was the result.